website/content/en/preview/concepts/_index.md

```diff
@@ -6,169 +6,105 @@ description: >
+Although most use cases are addressed with a single NodePool for multiple teams, multiple NodePools are useful to isolate nodes for billing, use different node constraints (such as no GPUs for a team), or use different disruption settings.
 
-### Deprovisioning nodes
+### Disrupting nodes
 
 Karpenter deletes nodes when they are no longer needed.
 
-* **Finalizer**: Karpenter places a finalizer bit on each node it creates.
+* [**Finalizer**]({{<ref "disruption#manual-methods" >}}): Karpenter places a finalizer bit on each node it creates.
 When a request comes in to delete one of those nodes (such as a TTL or a manual `kubectl delete node`), Karpenter will cordon the node, drain all the pods, terminate the EC2 instance, and delete the node object.
 Karpenter handles all clean-up work needed to properly delete the node.
-* **Node Expiry**: If a node expiry time-to-live value (`ttlSecondsUntilExpired`) is reached, that node is drained of pods and deleted (even if it is still running workloads).
-* **Empty nodes**: When the last workload pod running on a Karpenter-managed node is gone, the node is annotated with an emptiness timestamp.
-Once that "node empty" time-to-live (`ttlSecondsAfterEmpty`) is reached, finalization is triggered.
-* **Consolidation**: If enabled, Karpenter will work to actively reduce cluster cost by identifying when nodes can be removed as their workloads will run on other nodes in the cluster and when nodes can be replaced with cheaper variants due to a change in the workloads.
-* **Interruption**: If enabled, Karpenter will watch for upcoming involuntary interruption events that could affect your nodes (health events, spot interruption, etc.) and will cordon, drain, and terminate the node(s) ahead of the event to reduce workload disruption.
-
-For more details on how Karpenter deletes nodes, see [Deprovisioning nodes](./deprovisioning) for details.
-
-### Upgrading nodes
-
-A straight-forward way to upgrade nodes is to set `ttlSecondsUntilExpired`.
-Nodes will be terminated after a set period of time and will be replaced with newer nodes using the latest discovered AMI.
-See more in [AWSNodeTemplate](./node-templates).
-
-### Constraints
-
-The concept of layered constraints is key to using Karpenter.
-With no constraints defined in provisioners and none requested from pods being deployed, Karpenter chooses from the entire universe of features available to your cloud provider.
-Nodes can be created using any instance type and run in any zones.
+* [**Expiration**]({{<ref "disruption" >}}): Karpenter will mark nodes as expired and disrupt them after they have lived a set number of seconds, based on the NodePool's `spec.disruption.expireAfter` value. You can use node expiry to periodically recycle nodes due to security concerns.
```

Comment:
Probably better to use this anchorlink to refer to the text where expiration is talked about
```suggestion
* [**Expiration**]({{<ref "disruption#automated-methods" >}}): Karpenter will mark nodes as expired and disrupt them after they have lived a set number of seconds, based on the NodePool's `spec.disruption.expireAfter` value. You can use node expiry to periodically recycle nodes due to security concerns.
```

Reply:
Good call 👍